ocrd/ocrd/resource_manager.py

```diff
@@ -71,9 +70,7 @@ def list_installed(self, executable=None):
             # resources we know about
             all_executables = list(self.database.keys())
             # resources in the file system
-            parent_dirs = [join(x, 'ocrd-resources') for x in [XDG_CACHE_HOME, XDG_CONFIG_HOME, XDG_DATA_HOME]]
-            if VIRTUAL_ENV:
-                parent_dirs += [join(VIRTUAL_ENV, 'share', 'ocrd-resources')]
+            parent_dirs = [join(x, 'ocrd-resources') for x in [XDG_DATA_HOME, '/usr/local/share']]
```

Comment:
What about the CWD? I know it's probably not realistic, because at processor runtime we are in the workspace, whereas during `ocrd resmgr` we might not, but we could still try :-)

```suggestion
            parent_dirs = [join(x, 'ocrd-resources') for x in [XDG_DATA_HOME, '/usr/local/share', getcwd()]]
```

Reply:
Originally omitted because we didn't have the `ocrd-resources` directory and listing everything in PWD seemed overkill. But there's no reason now not include it.